app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
```

Comment:
The import statement is incorrect. The correct syntax should use a comma to separate multiple imports, like `from typing import Callable, Any`.